mkdocs.yml

```diff
@@ -62,22 +62,56 @@ pages:
   - Introduction: index.md
   - Using Atomist:
       - Getting Started: user/index.md
-      - GitHub: user/github.md
+      - Source Control: user/github.md
       - Slack: user/slack.md
-      - Continuous Integration: user/ci.md
+      - Built-in Chat Integrations: lifecycle.md
+      - Spring: user/spring.md
       - Kubernetes: user/kubernetes.md
   - Developer Quick Start: quick-start.md
   - Developer Guide:
-      - Overview: developer/index.md
-      - Prerequisites: developer/prerequisites.md
-      - Software Delivery Machine: developer/sdm.md
-      - SDM Project: developer/sdm-project.md
-      - Running SDMs: developer/run.md
-      - Creating Projects: developer/create.md
-      - Editing Projects: developer/edit.md
-      - Inspecting Projects: developer/inspect.md
+      - About this guide: developer/index.md
+      - Architecture: developer/architecture.md
+      - Running an SDM:
+        - About the SDM: developer/sdm.md
+        - Prerequisites: developer/prerequisites.md
+        - Team Mode: developer/team.md
+        - Local Mode: developer/local.md
+        - Deploying your SDM: developer/sdm-deploy.md
+      - Goals:
+          - About Goals: developer/goal.md
+          - Setting Goals: developer/set-goals.md
+          - Registrations: developer/registration.md
+      - Builds:
+        - Build Goal: developer/build.md
+        - Integrations: user/ci.md
+        - Reacting to Builds: developer/build-event.md
+      - Deploys: developer/deploy.md
+      - Code Inspections: developer/inspect.md
+      - Code Transform:
+          - Transforms: developer/transform.md
+          - Autofixes: developer/autofix.md
+          - Project: developer/project.md
       - Commands: developer/commands.md
```

Comment:
Didn't you also delete this file? How is the build succeeding with this still in the pages and the page not existing?

Reply:
Ah, you just deleted the contents.

Reply:
yep. They were too out of date. May be able to resurrect some with updated info, when we remake this page